test/e2e/e2e_suite_test.go

```diff
@@ -5,50 +5,41 @@ package e2e
 
 import (
 	"context"
-	"flag"
-	"fmt"
-	"os"
-	"os/user"
-	"path/filepath"
-	"strconv"
 	"testing"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/open-cluster-management/governance-policy-framework/test/common"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/dynamic"
 	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/klog"
 )
 
 var (
-	userNamespace          string
-	clusterNamespace       string
-	clientHub              kubernetes.Interface
-	clientHubDynamic       dynamic.Interface
-	clientManaged          kubernetes.Interface
-	clientManagedDynamic   dynamic.Interface
-	gvrPod                 schema.GroupVersionResource
-	gvrPolicy              schema.GroupVersionResource
-	gvrIamPolicy           schema.GroupVersionResource
-	gvrCertPolicy          schema.GroupVersionResource
-	gvrConfigurationPolicy schema.GroupVersionResource
-	gvrPlacementBinding    schema.GroupVersionResource
-	gvrPlacementRule       schema.GroupVersionResource
-	gvrRole                schema.GroupVersionResource
-	gvrCRD                 schema.GroupVersionResource
-	gvrNS                  schema.GroupVersionResource
-	gvrK8sRequiredLabels   schema.GroupVersionResource
-	kubeconfigHub          string
-	kubeconfigManaged      string
-	defaultTimeoutSeconds  int
+	clusterNamespace = "managed"
 
-	defaultImageRegistry string
+	defaultTimeoutSeconds int
+	userNamespace         string
+	clientHub             kubernetes.Interface
+	clientHubDynamic      dynamic.Interface
+	clientManaged         kubernetes.Interface
+	clientManagedDynamic  dynamic.Interface
+
+	gvrPod                 = schema.GroupVersionResource{Version: "v1", Resource: "pods"}
+	gvrNS                  = schema.GroupVersionResource{Version: "v1", Resource: "namespaces"}
+	gvrRole                = schema.GroupVersionResource{Group: "rbac.authorization.k8s.io", Version: "v1", Resource: "roles"}
+	gvrCRD                 = schema.GroupVersionResource{Group: "apiextensions.k8s.io", Version: "v1beta1", Resource: "customresourcedefinitions"}
+	gvrPolicy              = schema.GroupVersionResource{Group: "policy.open-cluster-management.io", Version: "v1", Resource: "policies"}
```

Comment:
Optional: What do you think about putting these GVRs in the `common` package? I noticed this one is redefined in `policy-collection` and I think it'd be useful in general to have these for other test packages in the future.

Reply:
Yeah - the question becomes which ones to include in `common` (all? *any* that are shared?), and is it worth it to update all the times these appear to prepend `common.<foo>`.

In my opinion, it wasn't necessary yet, but with another set of tests maybe we'll find a better set of commonly used GVRs

Reply:
@JustinKuli I was thinking it might be nice to just have them all in a separate file in the `common` package? They don't necessarily need to be used by multiple tests yet, but it sets up a convention to reduce duplication in the future.

Reply:
That sounds reasonable, I'll do that!